docs/src/layouts/links.js

```diff
@@ -487,11 +487,13 @@ export default [
   {
     id: 'USING_BACKPACK',
     category: 'Using Backpack',
+    sort: true,
```

Comment:
I changed these as it was bugging me it wasn't in alphabetical order (please don't be mad 😆 )

Reply:
omg how could you 😂